notify.js

```diff
@@ -130,9 +132,11 @@
     Notify.prototype.show = function() {
         this.myNotify = new N(this.title, {
             'body': this.options.body,
-            'tag' : this.options.tag,
-            'icon' : this.options.icon,
-            'lang' : this.options.lang
+            'tag': this.options.tag,
+            'icon': this.options.icon,
+            'lang': this.options.lang,
+            'requireInteraction': this.options.requireInteraction,
+            'closeOnClick': this.options.closeOnClick
         });
 
         if (this.options.timeout && !isNaN(this.options.timeout)) {
```

Comment:
I think we need to add a check for `requireInteraction` here, else if someone passes a timeout value it will override the option and close the notification automatically.